modal/client.py

```diff
@@ -412,6 +405,37 @@ def name(self) -> str:
         return self.wrapped_method.name
 
     async def __call__(
+        self,
+        req: RequestType,
+        base_delay: float = 0.1,
+        max_delay: float = 1,
+        delay_factor: float = 2,
+        max_retries: Optional[int] = 3,
+        additional_status_codes: list = [],
+        attempt_timeout: Optional[float] = None,
+        total_timeout: Optional[float] = None,
+        attempt_timeout_floor=2.0,
+        retry_warning_message: Optional[RetryWarningMessage] = None,
```

Comment:
Or alternatively, what would you think about packaging up the retry parameterization into a dataclass so the stub method signatures are more like `(request, metadata, retries)`? Feels like that could keep things organized a little better maybe?

Reply:
I wonder if that would also let us spell the no-retry case something like `stub.MyMethod(request, retries=None)` which is maybe a little bit more obvious than the `.direct()` suffix (although I don't hate that).